### ISSUE TITLE
Get a separate certificate scoped to subdomains with very weak settings?

### ISSUE THREAD

**Issue by lgarron** [chromium/badssl.com#230]
e.g. SSLv2, export crypto, 3DES

**Comment by marumari** [comment]
*.very.badssl.com?

On Dec 14, 2016, at 8:17 PM, Lucas Garron <notifications@github.com> wrote:

e.g. SSLv2, export crypto, 3DES

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/chromium/badssl.com/issues/230>, or mute the thread

**Comment by lgarron** [comment]
> *.very.badssl.com?

Yeah, we'd probably want the bad settings to handle selecting the sketchy cert.

In other news, very.badssl.com will have to move off SHA-1 soon to be accessible in Chrome. 😢

**Comment by marumari** [comment]
Poor doge!  :dog:
